src/createAnimatedComponent.js

```diff
@@ -396,6 +396,9 @@ export default function createAnimatedComponent(Component) {
         } else if (key === 'animatedProps') {
           Object.keys(value.initial).forEach((key) => {
             props[key] = value.initial[key];
+            if (value.viewRef?.current === null) {
```

Comment:
`value.viewRef` is optional in `animatedProps`? I'm asking because `style` prop uses `style.viewRef.current === null`

Reply:
Right, in `createAnimatedComponent` the `viewRef` should always exists, but in updateProps is [optional](https://github.com/software-mansion/react-native-reanimated/blob/master/src/reanimated2/Hooks.js#L449).